### ISSUE TITLE
pod install conflict 

### ISSUE THREAD

**Issue by athlona64** [rmtmckenzie/flutter_qr_mobile_vision#136]
this error happen when change qr_mobile_vision to latest version if revert to old version is not a problem but can't upload to apple store

```[!] CocoaPods could not find compatible versions for pod "Firebase/Analytics":
  In Podfile:
    Firebase/Analytics

    firebase_analytics (from `.symlinks/plugins/firebase_analytics/ios`) was resolved to 0.0.1, which depends on
      Firebase/Analytics (~> 6.0)

CocoaPods could not find compatible versions for pod "Firebase/Auth":
  In Podfile:
    Firebase/Auth

    firebase_auth (from `.symlinks/plugins/firebase_auth/ios`) was resolved to 0.0.1, which depends on
      Firebase/Auth (~> 6.3)

CocoaPods could not find compatible versions for pod "Firebase/Core":
  In Podfile:
    firebase_analytics (from `.symlinks/plugins/firebase_analytics/ios`) was resolved to 0.0.1, which depends on
      Firebase/Analytics (~> 6.0) was resolved to 6.3.0, which depends on
        Firebase/Core (= 6.3.0)

    firebase_core (from `.symlinks/plugins/firebase_core/ios`) was resolved to 0.0.1, which depends on
      Firebase/Core

CocoaPods could not find compatible versions for pod "Firebase/CoreOnly":
  In Podfile:
    firebase_auth (from `.symlinks/plugins/firebase_auth/ios`) was resolved to 0.0.1, which depends on
      Firebase/Auth (~> 6.3) was resolved to 6.3.0, which depends on
        Firebase/CoreOnly (= 6.3.0)

    qr_mobile_vision (from `.symlinks/plugins/qr_mobile_vision/ios`) was resolved to 0.0.1, which depends on
      Firebase/MLVisionBarcodeModel was resolved to 5.0.0, which depends on
        Firebase/CoreOnly (= 5.0.0)

CocoaPods could not find compatible versions for pod "FirebaseCore":
  In Podfile:
    cloud_firestore (from `.symlinks/plugins/cloud_firestore/ios`) was resolved to 0.0.1, which depends on
      Firebase/Firestore (~> 6.0) was resolved to 6.3.0, which depends on
        FirebaseFirestore (~> 1.4.0) was resolved to 1.4.5, which depends on
          FirebaseCore (~> 6.2)

    firebase_auth (from `.symlinks/plugins/firebase_auth/ios`) was resolved to 0.0.1, which depends on
      Firebase/Auth (~> 6.3) was resolved to 6.3.0, which depends on
        Firebase/CoreOnly (= 6.3.0) was resolved to 6.3.0, which depends on
          FirebaseCore (= 6.0.3)

    firebase_auth (from `.symlinks/plugins/firebase_auth/ios`) was resolved to 0.0.1, which depends on
      Firebase/Auth (~> 6.3) was resolved to 6.3.0, which depends on
        FirebaseAuth (~> 6.1.2) was resolved to 6.1.2, which depends on
          FirebaseCore (~> 6.0)
```

pubspec.yaml
```
name: xxxx
description: A new Flutter application.

# The following defines the version and build number for your application.
# A version number is three numbers separated by dots, like 1.2.43
# followed by an optional build number separated by a +.
# Both the version and the builder number may be overridden in flutter
# build by specifying --build-name and --build-number, respectively.
# In Android, build-name is used as versionName while build-number used as versionCode.
# Read more about Android versioning at https://developer.android.com/studio/publish/versioning
# In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
# Read more about iOS versioning at
# https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html



version: 1.0.76+76





environment:
  sdk: ">=2.3.0 <3.0.0"

dependencies:
  flutter_platform_widgets: ^0.20.3-beta.1
  path_provider: ^1.3.0
  flutter:
    sdk: flutter
  permission: ^0.1.3
  share: ^0.6.3+6
  fluttertoast: ^3.1.3
  shared_preferences: 0.4.3
  flutter_screenutil: ^0.5.2
  qr_flutter: ^3.0.1
  web3dart: ^1.1.1+1
  web_socket_channel: ^1.0.8
  bip39: ^1.0.2
  bitcoin_flutter: ^2.0.0
  toast: ^0.1.3
  crypto: ^2.1.4
  flutter_spinkit: ^4.1.1+1
  after_layout: ^1.0.7+2
  cached_network_image: ^2.0.0
  intl: ^0.16.0
#  firebase_ml_vision: ^0.9.3+8
  barcode_scan: ^3.0.1
  ethereum: 4.0.1
  bmnav: 0.3.4
  loading_indicator: ^1.0.0
  image_gallery_saver: ^1.2.2
  firebase_storage: ^3.1.1
  package_info: ^0.4.0+13
  native_widgets: ^1.3.5

    # The following adds the Cupertino Icons font to your application.
    # Use with the CupertinoIcons class for iOS style icons.
  cupertino_icons: ^0.1.2
  modal_progress_hud: ^0.1.2
  http: ^0.12.0+2
  auto_size_text: ^2.0.1
  flutter_file_manager: ^0.2.0
  contacts_service: ^0.0.8
  pattern_formatter: ^1.0.1
  countdown: ^0.1.0
  english_words: ^3.1.0
  #man
  ed25519_hd_key: ^1.0.1
  dart_ssss: ^0.0.5
  local_auth: ^0.6.0+1
  get_it: ^3.0.1
  passcode_screen:
    path: ./passcode_screen-1.0.2
  firebase_auth: 0.15.3+1
  firebase_core: 0.4.4
  permission_handler: ^4.4.0+hotfix.2
  firebase_analytics: ^5.0.6
  pull_to_refresh: ^1.5.7
  cloud_firestore: ^0.12.11
  bip32_dart: ^1.0.5
  json_annotation: ^3.0.0
  provider: ^3.2.0
  flutter_share_me: ^0.9.0
  encrypt: ^4.0.0
  flutter_secure_storage: ^3.3.1+1
  ads: ^1.7.2
  image_picker: ^0.6.3+1
  http_parser: ^3.1.3
  dio: ^3.0.8
  infinite_listview: ^1.0.1+1
  qrscan: ^0.2.17
  rxdart: ^0.23.1
  keyboard_actions: ^3.2.0
  keyboard_visibility: ^0.5.6
  carousel_slider: ^1.3.0
  mqtt_client: ^6.1.0
  in_app_update: ^1.1.7
  url_launcher: ^5.4.2
  dash_chat: ^1.0.18
  avataaar_image: ^1.0.8
  #ios pod
#  flutter_camera_ml_vision: ^2.2.4
  camera: ^0.5.7+3
  qr_mobile_vision: ^1.0.1+1
  launch_review: ^2.0.0
  firebase_messaging: ^6.0.9
  flutter_local_notifications: ^1.2.0+2
  flare_flutter: ^1.8.3
  date_format: any
  webview_flutter: ^0.3.19+8
  #Opt-in to the embedded views preview by adding a boolean property to the app's Info.plist file with the key io.flutter.embedded_views_preview and the value YES.


#  flutter_money_formatter:
#    path: ./flutter_money_formatter
  snaplist: ^0.1.8
  sms_autofill: ^1.1.1

  flutter_localizations:
    sdk: flutter

dependency_overrides:
  crypto: 2.1.4

dev_dependencies:
  flutter_test:
    sdk: flutter

flutter_icons:
  ios: true
  image_path: assets/icon/icon.png

# For information on the generic Dart part of this file, see the
# following page: https://dart.dev/tools/pub/pubspec

# The following section is specific to Flutter.
flutter:

  # The following line ensures that the Material Icons font is
  # included with your application, so that you can use the icons in
  # the material Icons class.
  uses-material-design: true

  # To add assets to your application, add an assets section, like this:
  assets:
    #    - assets/images/g606.png
    #    - images/g606.png
  - assets/bip39.txt
  - assets/image/
  - assets/i18n/en.json
  - assets/i18n/th.json
  - assets/image/slot/
  - assets/i18n/vi.json
  - assets/i18n/km.json
  - assets/i18n/lo.json
  - assets/image/checkin/
  - assets/image/locklike/
  - assets/animation/checkin/
  - assets/animation/
  - assets/image/ads/

  - assets/

  #  - images/a_dot_burr.jpeg
  #  - images/a_dot_ham.jpeg

  # An image asset can refer to one or more resolution-specific "variants", see
  # https://flutter.dev/assets-and-images/#resolution-aware.

  # For details regarding adding assets from package dependencies, see
  # https://flutter.dev/assets-and-images/#from-packages

  # To add custom fonts to your application, add a fonts section here,
  # in this "flutter" section. Each entry in this list should have a
  # "family" key with the font family name, and a "fonts" key with a
  # list giving the asset and other descriptors for the font. For
  # example:
  fonts:
    - family: Proxima Nova
      fonts:
        - asset: assets/fonts/ProximaNova-Black.ttf
        - asset: assets/fonts/ProximaNova-Bold.ttf
        - asset: assets/fonts/ProximaNova-Extrabld.ttf
        - asset: assets/fonts/ProximaNova-Light.ttf
        - asset: assets/fonts/ProximaNova-Regular.ttf
        - asset: assets/fonts/ProximaNova-Semibold.ttf
    - family: Proxima Nova Thin
      fonts:
        - asset: assets/fonts/ProximaNovaT-Thin.ttf
    - family: Roboto
      fonts:
        - asset: assets/fonts/Roboto-Regular.ttf
          weight: 100
        - asset: assets/fonts/Roboto-Bold.ttf
          weight: 700
    - family: MyFlutterApp
      fonts:
        - asset: assets/fonts/MyFlutterApp.ttf
    - family: icon_home
      fonts:
        - asset: assets/fonts/icon_home.ttf
    - family: Mining
      fonts:
        - asset: assets/fonts/Mining.ttf
    - family: Spendlike
      fonts:
        - asset: assets/fonts/Spendlike.ttf
  # For details regarding fonts from package dependencies,
  # see https://flutter.dev/custom-fonts/#from-packages

```

**Comment by aiman5252** [comment]
Hi, I pretty much got the same error too.
I usually debug the android version of this app which already integrated with firebase, and when I tried to run on simulator this error happens.
*note: I haven't integrated the iOS version with firebase yet.

```[!] CocoaPods could not find compatible versions for pod "Firebase/CoreOnly":
  In snapshot (Podfile.lock):
    Firebase/CoreOnly (= 6.14.0)

  In Podfile:
    firebase_analytics (from `.symlinks/plugins/firebase_analytics/ios`) was resolved to 0.0.1, which depends on
      Firebase/Core was resolved to 6.14.0, which depends on
        Firebase/CoreOnly (= 6.14.0)

    qr_mobile_vision (from `.symlinks/plugins/qr_mobile_vision/ios`) was resolved to 0.0.1, which depends on
      Firebase/MLVisionBarcodeModel was resolved to 5.0.0, which depends on
        Firebase/CoreOnly (= 5.0.0)


You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * changed the constraints of dependency `Firebase/CoreOnly` inside your development pod `qr_mobile_vision`.
   You should run `pod update Firebase/CoreOnly` to apply changes you've made.
CocoaPods could not find compatible versions for pod "GTMSessionFetcher/Core":
  In snapshot (Podfile.lock):
    GTMSessionFetcher/Core (= 1.3.0, ~> 1.1)

  In Podfile:
    qr_mobile_vision (from `.symlinks/plugins/qr_mobile_vision/ios`) was resolved to 0.0.1, which depends on
      Firebase/MLVision was resolved to 6.14.0, which depends on
        FirebaseMLVision (~> 0.19.0) was resolved to 0.19.0, which depends on
          GTMSessionFetcher/Core (~> 1.1)

    qr_mobile_vision (from `.symlinks/plugins/qr_mobile_vision/ios`) was resolved to 0.0.1, which depends on
      Firebase/MLVision was resolved to 6.14.0, which depends on
        FirebaseMLVision (~> 0.19.0) was resolved to 0.19.0, which depends on
          GoogleAPIClientForREST/Core (~> 1.3) was resolved to 1.4.1, which depends on
            GTMSessionFetcher (>= 1.1.7) was resolved to 1.3.1, which depends on
              GTMSessionFetcher/Full (= 1.3.1) was resolved to 1.3.1, which depends on
                GTMSessionFetcher/Core (= 1.3.1)


You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * changed the constraints of dependency `GTMSessionFetcher/Core` inside your development pod `qr_mobile_vision`.
   You should run `pod update GTMSessionFetcher/Core` to apply changes you've made.```

Here's the list of things that I've done:
- implement FirebaseApp.configure() on AppDelegate
- trying to integrate iOS version of the app with Firebase but unsuccessful because I can't run it
- `pod remove trunk` and `pod install`

**Comment by rmtmckenzie** [comment]
I'm closing this as a duplicate of #129.
